docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -11,11 +11,63 @@ Before consolidating a new QoS Policy, it is usually set using this generic QoS
 Consequently, this section is prone to frequent updates so the user is advised to check latest changes after upgrading
 to a different release version.
 
-``PushMode`` QoS Policy
-^^^^^^^^^^^^^^^^^^^^^^^
+.. _push_mode_qos_policy:
+
+``Push Mode`` QoS Policy
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, Fast DDS DataWriters are enabled using the ``push mode``.
+This implies they will send new samples to matched readers immediately after adding them to their queue.
```

Comment:
Their queue here means datawriters', but it could be confused with datareaders'
```suggestion
This implies that they will send new samples to the matched |DataReaders| immediately after adding them to their queue.
```

Reply:
Done on e3a513774fc11e066ef7c8e05e7a6cce7505374d